projects/components/src/dropdown/dropdown.component.ts

```diff
@@ -88,7 +109,39 @@ export class DropdownComponent<T extends DropdownItem<T>> {
         return this._items;
     }
 
-    constructor(@Optional() @SkipSelf() @Host() private parentVcdDropdown: DropdownComponent<T>) {}
+    @Output() dropdownMenuUpdated = new EventEmitter<{ menu: HTMLElement }>();
+
+    constructor(@Optional() @SkipSelf() private parentVcdDropdown: DropdownComponent<T>) {}
+    @ViewChild(ClrDropdownMenu)
+    set clrDropdownMenu(val: ClrDropdownMenu) {
+        this._clrDropdownMenu = val;
+        if (!val) {
+            return;
+        }
+        // Disable Claritys focus handling logic
+        this._clrDropdownMenu.items.reset([]);
+        this._clrDropdownMenu.items.notifyOnChanges();
+    }
+    private _clrDropdownMenu: ClrDropdownMenu;
+
+    @ViewChild(ClrDropdownTrigger, { read: ElementRef, static: true })
+    set dropdownTriggerEl(val: ElementRef<HTMLElement>) {
+        this._dropdownTriggerEl = val ? val.nativeElement : null;
+    }
+    _dropdownTriggerEl: HTMLElement;
```

Comment:
private?

Reply:
It is being accessed from the outside in dropdown-focus-handler.directive.ts